### PR TITLE
[DOCS] Add `Databricks (SQL)` to Cloud supported list

### PR DESCRIPTION
**Pull request by Kilo59**
Move `Databricks (SQL)` to a "Cloud Supported" datasource.

### CODE REVIEW

File: docs/docusaurus/docs/application_integration_support.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The following table defines the GX Cloud, GX Core, and Community Supported integ
 
 | Integration Type                 | GX Cloud                              | GX Core                                                                                    | Community            |
 | -------------------------------- | ------------------------------------- | ------------------------------------------------------------------------------------------ | -------------------- |
-| Data Sources<sup>1</sup>         | Snowflake<br/> PostgreSQL<sup>2</sup> | Snowflake<br/>PostgreSQL<br/>Sqlite<br/>Databricks (SQL)<br/>BigQuery<br/>Spark<br/>Pandas | MSSQL<br/>MySQL<br/> |
+| Data Sources<sup>1</sup>         | Snowflake<br/>Databricks (SQL)<br/> PostgreSQL<sup>2</sup> | Snowflake<br/>PostgreSQL<br/>Sqlite<br/>BigQuery<br/>Spark<br/>Pandas | MSSQL<br/>MySQL<br/> |
```
Comment (rachhouse) [inline docs/docusaurus/docs/application_integration_support.md]: Is Databricks SQL supported in GX Core as well? If so, let's add it to the table too.

Comment (Kilo59) [reply docs/docusaurus/docs/application_integration_support.md]: My mistake. Good catch.

Comment (Kilo59) [inline docs/docusaurus/docs/application_integration_support.md]:
```suggestion
| Data Sources<sup>1</sup>         | Snowflake<br/>Databricks (SQL)<br/> PostgreSQL<sup>2</sup> | Snowflake<br/>Databricks (SQL)<br/>PostgreSQL<br/>Sqlite<br/>BigQuery<br/>Spark<br/>Pandas | MSSQL<br/>MySQL<br/> |
```

**Review by rachhouse — APPROVED** [review]
Looks good to me!

**Pull request merged**